sdk/core/core-amqp/rollup.test.config.js

```diff
@@ -1,7 +1,60 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+import cjs from "@rollup/plugin-commonjs";
+import inject from "@rollup/plugin-inject";
+import json from "@rollup/plugin-json";
+import multiEntry from "@rollup/plugin-multi-entry";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import shim from "rollup-plugin-shim";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import { makeConfig, makeBrowserTestConfig } from "@azure/dev-tool/shared-config/rollup";
 
-import * as base from "./rollup.base.config";
+const inputs = makeConfig(require("./package.json"));
 
-// Node tests are run via ts-node
-export default [base.browserConfig(true)];
+if (!process.env.ONLY_NODE) {
+  // service-bus has many dependencies that we do not
+  // want to bring into the shared rollup config, so
+  // replace the original test config with a patched one
+  inputs[1] = makeBrowserTestConfigPatch();
+}
+
+function makeBrowserTestConfigPatch() {
+  const config = { ...makeBrowserTestConfig(require("./package.json")) };
+  config.plugins = [
+    multiEntry({ exports: false }),
+    sourcemaps(),
+    replace({
+      delimiters: ["", ""],
+    }),
+    // fs, net, and tls are used by rhea and need to be shimmed
+    // dotenv doesn't work in the browser, so replace it with a no-op function
+    shim({
+      dotenv: `export function config() { }`,
```

Comment:
I think browser tests should not have dotenv uses that need to be shimmed

Reply:
You are right.  We actually don't use env vars at all in any of the tests.  Removing.